web/styles/input_pill.css

```diff
@@ -119,6 +119,19 @@
             perspective: 1000px;
         }
     }
+
+    &.not-editable-by-user {
+        cursor: not-allowed;
+        background-color: hsl(0deg 0% 93%);
+
+        .pill {
+            cursor: not-allowed;
+        }
+
+        .exit {
+            display: none;
+        }
+    }
```

Comment:
Why are we not using `.not-editable` class? Many properties applied here are already defined there.

Reply:
This was also suggested in previous PR #23852. I agree with the reasoning provided there

> If we use not-editable class then we can avoid most of the css but would have to specify padding , border and background-color in disabled class in order to make it consistent with other types of fileds in disabled look.

Reply:
Okay got it.